packages/web-component/src/components/read-along-component/read-along.tsx

```diff
@@ -208,11 +210,12 @@ export class ReadAlongComponent {
    */
   private urlTransform(path: string): string {
     if (
-      this.useAssetsFolder &&
+      this.imageAssetsFolder &&
       looksLikeRelativePath(path) &&
       !path.startsWith("blob")
-    )
-      return "assets/" + path;
+    ) {
+      return this.imageAssetsFolder + path;
```

Comment:
As mentioned above, that's a guaranteed gotcha. The intuitive thing to do in the read-along element is `image-assets-folder="images"` and that's going to generate e-mail enquiries for support.
I'd rather implement the logic 
 - if folder is empty or undefined return path
 - else return folder + "/" + path
This else logic is safe even if the user adds the slash, because doubling the slash won't break the URL.

Reply:
In `async componentWillLoad()` you can check for the trailing slash and add if needed
`
if (!this.imageAssetsFolder.endsWith("/")){
      this.imageAssetsFolder+="/"
    }
`